vault/README.md

```diff
@@ -26,6 +26,43 @@ For containerized environments, see the [Autodiscovery Integration Templates][2]
 | `<INIT_CONFIG>`      | blank or `{}`                            |
 | `<INSTANCE_CONFIG>`  | `{"api_url": "http://%%host%%:8200/v1"}` |
 
+#### Log Collection
+
+**Available for Agent >6.0**
+
+1. Collecting logs is disabled by default in the Datadog Agent, enable it in your `datadog.yaml` file:
+
+```
+logs_enabled: true
+```
+
+2. Configure Vault to enable audit and server logs.
+    * Audit logs must enabled by a privileged user with the appropriate policies, [see more][11].
+        ```
+        $ vault audit enable file file_path=/vault/vault-audit.log
+        ```
+    * [Server logs][12] are not written to file by default. You can configure static server logs in the [start up script][13].
+        The following script is outputting the logs to `/var/log/vault.log`.
+        ```
+        ...
+        [Service]
+        ...
+        ExecStart=/bin/sh -c '/home/vagrant/bin/vault server -config=/home/vagrant/vault_nano/config/vault -log-level="trace" > /var/log/vault.log
+        ...
+        ```
+ 
+3. Add this configuration block to your `vault.d/conf.yaml` file to start collecting your Vault logs:
+    ````yaml
+    logs:
+    - type: file
+        path: /vault/vault-audit.log
+        source: vault
+        service: vault
+    - type: file
+        path: /var/log/vault.log
+        source: vault
```

Comment:
Should this block have `service: vault` as well?

Reply:
Let's add `service: <SERVICE_NAME>`